xml/art_solution_automation.xml

```diff
@@ -0,0 +1,438 @@
+            with this information.
+          </para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>&salt; Minions</term>
+        <listitem>
+          <para>A server or machine often controlled by the &saltmaster;.
+          Its main purpose is to execute commands sent from the &saltmaster;,
+          report data back, and send information about itself.
+          </para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>&salt; States</term>
+        <listitem>
+          <para>
+            YAML text file to maintain consistency across your environment.
+            &salt; states can be executed.
+          </para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>&terraform;</term>
+        <listitem>
+          <remark>toms 2020-06-10: https://www.terraform.io/intro/index.html</remark>
+          <para>
+            An <quote>infrastructure as code</quote> software tool.
+            It deploys the needed infrastructure in the cloud or virtual environments
+            and &ay; for on-premises deployments.
+          </para>
+          <para>
+            Currently, &terraform; is <emphasis role="strong">NOT</emphasis> supported
+            by &suse;.
+          </para>
+        </listitem>
+      </varlistentry>
+    </variablelist>
+  </sect1>
+
+  <sect1 xml:id="sec-sol-autom-using-salt-formulas">
+    <title>Using &salt; Formulas</title>
+    <remark>toms 2020-06-29: Is there a general way to install/use these? Should
+    we tell our readers about that?</remark>
+    <para>
+      To use &salt; formulas from &suse;, execute the following steps:
+    </para>
+    <!--
+    <procedure>
+      <step>
+        <para>Install the </para>
+        <screen>git clone https://github.com/SUSE/shaptools.git
+cd shaptools
+sudo python setup.py install</screen>
+      </step>
+      <step>
+        <para></para>
+      </step>
+    </procedure>
+    -->
+  </sect1>
+
+  <sect1 xml:id="sec-sol-autom-hana-deploy">
+    <title>&hana; Automated Deployment</title>
+    <para>
+      &salt; formula <package>saphanabootstrap-formula</package> for bootstrapping
+      and managing &hana; platform and system replication.
+    </para>
+    <formalpara>
+      <title>Link:</title>
+      <para>See <link xlink:href="https://github.com/SUSE/saphanabootstrap-formula"/>.</para>
+    </formalpara>
+    <itemizedlist>
+      <title>Capabilities for <package>saphanabootstrap-formula</package></title>
+      <listitem>
+        <para>
+          Installs &hana;.
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Configures system replication.
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Configures &ha; cluster.
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Configures optional monitor exporter.
```

Comment:
I would rewrite this to: `Configures the SAP HANA prometheus exporter`. We could even paste here the link to the exporter project

Reply:
Ok, added. As Prometheus is a name, I used a capital "P". :wink: